mslib/msui/mscolab.py

```diff
@@ -51,9 +52,12 @@
 from mslib.msui import mscolab_admin_window as maw
 from mslib.msui import mscolab_version_history as mvh
 from mslib.msui import socket_control as sc
+from mslib.mscolab.file_manager import FileManager
```

Comment:
keep in mind the UI should not do anything with the servers code

On a future release we also separate the server resources from the client resources.